open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconChooserPanel.java

```diff
@@ -149,6 +149,12 @@
     /** The optional selected icon URL. */
     private String mySelectedUrl;
 
+    /** The icon display width */
+	public int myTileWidth;

```

Comment:
Since myTileWidth is only used by the BuildIconGridWorker, it'd be better to pass that in its constructor, and get rid of it here.  That way less state to manage, and this class stays simpler.

Reply:
resolved